samples/python/beam_search_causal_lm/README.md

```diff
@@ -6,7 +6,7 @@ This example showcases inference of text-generation Large Language Models (LLMs)
 
 The `--upgrade-strategy eager` option is needed to ensure `optimum-intel` is upgraded to the latest version.
 
-It's not required to install [../../requirements.txt](../../requirements.txt) for deployment if the model has already been exported.
+It's not required to install [../../requirements.txt](../../requirements.txt) for deployment if the model has already been exported. `numpy`, `openvino`, and `openvino-genai` are required for python deployment.
```

Comment:
can we just mention openvino-genai as it depends on OpenVINO which depends on numpy ?

Reply:
That way `pip install openvino_genai --no-dependencies` would be declared a valid way.

Reply:
`pip install openvino_genai --no-dependencies`

why do we need to suppose such case? Typically, people don't care about dependencies of dependencies